deltaspike/parent/code/pom.xml

```diff
@@ -417,6 +417,139 @@
                 </dependency>
             </dependencies>
         </profile>
+        
+        <profile>
+            <!-- use this profile to compile and test DeltaSpike with JBoss Weld 3.x -->
+            <id>Weld3</id>
```

Comment:
OK, I found a way: weld-core-bom is actually a "depchain" and not really a BOM, so we can keep just one Weld profile and by just importing weld-core-bom we can omit all other dependencies (weld-core/weld-core-impl, etc). I'll update the PR.